DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: c14bazAAR
 Title: Download and Prepare C14 Dates from Different Source Databases
 Description: Query different C14 date databases and apply basic data cleaning, merging and calibration steps. Currently available databases: 14cpalaeolithic, 14sea, adrac, austarch, calpal, context, emedyd, eubar, euroevol, irdd, jomon, katsianis, kiteeastafrica, medafricarbon, mesorad, pacea, palmisano, radon, radonb.
-Version: 2.0.0
+Version: 2.0.1
```

Comment:
I think we can probably finish both #127 and #131 and then directly go to 2.1.0. New databases are new features and should therefore probably increment the middle version number.

Reply:
> Why are there changes in the binary data/material_thesaurus.rda file, but apparently not in the underlying .csv file? 🤔

Running `data-raw/data_prep.R` generates a new .rda , does it not?